docs/docker-stack/entrypoint.rst

```diff
@@ -25,68 +25,6 @@ In some cases, you can pass environment variables to the image to trigger some o
 The variables that control the "execution" behaviour start with ``_AIRFLOW`` to distinguish them
 from the variables used to build the image starting with ``AIRFLOW``.
 
-The image entrypoint works as follows:
```

Comment:
This section contains a lot of information, which is repeated again in the sections below.